lib/verifier/hathifiles_contents_verifier.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+require "zlib"
+require_relative "../verifier"
+
+# Verifies that hathifiles workflow stage did what it was supposed to.
+
+module PostZephirProcessing
+  class HathifileContentsVerifier < Verifier
+    HATHIFILE_FIELDS_COUNT = 26
+    HATHIFILE_FIELD_REGEXES = [
+      # htid - required; lowercase alphanumeric namespace, period, non-whitespace ID
+      /^[a-z0-9]{2,4}\.\S+$/,
```

Comment:
Use:
```ruby
{ name: 'htid', regex: /^[a-z0-9]{2,4}\.\S+$/ }
```

Remove: `HATHIFILE_FIELDS_COUNT` -- duplicative of just counting how many fields there are in the regex specifications